lib/msf/core/handler/reverse_http.rb

```diff
@@ -355,6 +357,20 @@ def bind_port
     port > 0 ? port : datastore['LPORT'].to_i
   end
 
+  def get_ssl_cert_hash
```

Comment:
Does this make sense for reverse_http?  There's no SSL happening in the stager in this case.